dev/com.ibm.ws.ejbcontainer.bindings_fat/test-applications/BindingNameWeb.war/src/com/ibm/ejb3x/BindingName/web/BindingNameTestServlet.java

```diff
@@ -449,46 +420,81 @@ private void testLookupCombinationsHelper(Context context, String contextString,
         }
     }
 
+    private void testLookupCombinationsHelperRemote(Context context, String contextString, String lookupName, String BindingName, String beanNum, String passingCases) {
+        try {
+            System.out.println("Testing " + lookupName + " with context " + contextString + " against " + BindingName);
+            RemoteBindingNameIntf beanHome = (RemoteBindingNameIntf) context.lookup(lookupName);
+            if (passingCases.contains(beanNum)) {
+                if (context.lookup(lookupName) == null) {
+                    fail("lookup " + lookupName + " should have worked for " + BindingName + " and context " + contextString);
+                }
+                try {
+                    if (beanHome.foo() == null) {
+                        fail("bean.foo() for lookup " + lookupName + " should have worked for " + BindingName + " and context " + contextString);
+                    }
+                } catch (Exception e) {
+                    fail("bean.foo() for lookup " + lookupName + " should have worked for " + BindingName + " and context " + contextString);
```

Comment:
For any "fail" that is in a catch block, it would be nice if the fail message include the exception message.  May help in future diagnosis of a problem.